src/screens/AddSiteFormScreen.js

```diff
@@ -109,628 +59,25 @@ class AddSiteFormScreen extends Component {
-                        selectedValue={accessibilityOption}
-                        onValueChange={this.onUpdateAccessibilityOption}
-                    >
-                        {this.accessibilityOptions()}
-                    </Picker>
-
-                    <Text style={labelStyle}>{price}</Text>
-                    <Picker
-                        style={[formInputStyle, pickerStyle]}
-                        selectedValue={priceOption}
-                        onValueChange={this.onUpdatePriceOption}
-                    >
-                        {this.priceOptions()}
-                    </Picker>
-
-                    <Text style={labelStyle}>{facilities}</Text>
-                    <View style={formInputStyle}>
-                        {this.renderCheckboxes(facilities_options)}
-                    </View>
-
-                    <Text style={labelStyle}>{features}</Text>
-                    <View style={formInputStyle}>
-                        {this.renderCheckboxes(features_options)}
-                    </View>
-
-                    <Text style={labelStyle}>{alternate_sites} {optional}</Text>
-                    <Input
-                        containerStyle={formInputStyle}
-                        placeholder={alternate_sites_placeholder}
-                        value={siteAlternateSitesText}
-                        onChangeText={this.onUpdateAlternateSitesText}
-                        containerStyle={largeTextInput}
-                        blurOnSubmit={true}
-                        multiline={true}
-                        autoGrow={true}
-                        editable={true}
-                    />
-
-                    <Text style={labelStyle}>{cell_service} {optional}</Text>
-                    <View style={cellServiceContainerStyle}>
-                        <Picker
-                            style={[formInputStyle, cellProviderPickerStyle]}
-                            selectedValue={cellProviderOption}
-                            onValueChange={this.onUpdateCellProviderOption}
-                        >
-                            {this.cellProviderOptions()}
-                        </Picker>
-
-                        <Picker
-                            style={[formInputStyle, cellStrengthPickerStyle]}
-                            selectedValue={cellStrengthOption}
-                            onValueChange={this.onUpdateCellStrengthOption}
-                        >
-                            {this.cellStrengthOptions()}
-                        </Picker>
-                    </View>
-
-                    {this.renderCountyOptions()}
-
-                    {this.renderForestOptions()}
-
-                    {this.renderMVUMOptions()}
-
-                    {this.renderSubmitButton()}
-                </View>
-            </KeyboardAwareScrollView>
+            <SiteInfoInputForm
+                siteFormType={site_form_type.ADD}
+            />
```

Comment:
This whole file is now just a small call to the `SiteInfoInputForm` with the type of form we need.